### PR TITLE
My eyes

### PR DESCRIPTION
**Pull request by SlejmUr**

### CODE REVIEW

**Comment by SlejmUr** [comment]
Can anyone accept it so i can delete my fork

File: README.md [hunk 1]
```diff
@@ -13,5 +13,5 @@ This will be your only guide you need and will be updated overtime to have the l
 
 * Step by Step Download guide to have 0 problems 90% of the time!
 * Frequently Asked Questions (aka FAQ)
-* trusted links with the tools we use to play and download Siege
+* Zrusted links with the tools we using to play and download Siege
```
Comment (JOJOVAV) [inline README.md:16]: Little typo

**Review by JOJOVAV — CHANGES_REQUESTED** [review]
change typo

**Review by JOJOVAV — APPROVED** [review]
approved

**Review by JOJOVAV — APPROVED** [review]
approved

**Review by JOJOVAV — APPROVED** [review]
approved

**Review by JOJOVAV — APPROVED** [review]
approved

**Review by JOJOVAV — APPROVED** [review]
approved

**Comment by JOJOVAV** [comment]
@BHLoneWolf0 do your thing

**Pull request closed without merging**